packages/ded/elasticsearch/ingest_pipeline/ml_ded_ingest_pipeline.yml

```
@@ -0,0 +1,17 @@
+---
+description: Pipeline for the Data Exfiltration Detection Integration transform.
+processors:
+  - script:
+      description: Casts the value of the field `avg_source_bytes` to a long and assigns it to `source.bytes`.
+      lang: painless
+      source: |
+        if (ctx.avg_source_bytes != null) {
+            ctx['source']['bytes'] = (long)(ctx.avg_source_bytes);
+        }
```

```suggestion
  - convert: 
      field: avg_source_bytes
      type: long
      ignore_missing: true
  - remove:
      field: avg_source_bytes
      ignore_missing: true
```

This can be accomplished with `convert`. Also, do we need to retain `avg_source_bytes` for anything?

---

No, we do not. I will verify that this works correctly with convert and try it in-place.